dahu/core/app/scripts/views/workspace/actions/move.js

```diff
@@ -5,19 +5,49 @@
 define([
     'handlebars',
     'backbone.marionette',
+    // views
+    'views/workspace/actions/action',
     // templates
     'text!templates/views/workspace/actions/move.html'
 ], function(
     Handlebars,
     Marionette,
+    // views
+    ActionView,
     // templates
     moveTemplate
 ) {
-    
+
+    var ExtraParamView = Marionette.ItemView.extend({
+
+        template: Handlebars.default.compile(moveTemplate),
+
+        initialize: function (model) {
+            this.model = model;
+        },
+
+        triggers: {
+            "change .trXChoice": "move:trx:change",
+            "change .trYChoice": "move:try:change"
+        }
+    });
+
     /**
      * Move action view
      */
-    return Marionette.ItemView.extend({
-        template: Handlebars.default.compile(moveTemplate)
+    return ActionView.extend({
+
+        onChildviewMoveTrxChange: function () {
+            this.model.set("trX", $("#trXChoice_" + this.model.id).val());
+        },
+
+        onChildviewMoveTryChange: function () {
+            this.model.set("trY", $("#trYChoice_" + this.model.id).val());
+        },
+
+        onRender: function () {
```

Comment:
put it in onBeforeShow (see https://github.com/marionettejs/backbone.marionette/blob/master/docs/marionette.region.md#events-raised-during-show)